std/ascii.zig

```diff
@@ -1,29 +1,31 @@
 // Does NOT look at the locale the way C89's toupper(3), isspace() et cetera does.
-// I could have taken only a u7 to make this clear, but it would be slower
 // It is my opinion that encodings other than UTF-8 should not be supported.
 //
 // (and 128 bytes is not much to pay).
 // Also does not handle Unicode character classes.
 //
 // https://upload.wikimedia.org/wikipedia/commons/thumb/c/cf/USASCII_code_chart.png/1200px-USASCII_code_chart.png
 
-const tIndex = enum(u3) {
-    Alpha,
-    Hex,
-    Space,
-    Digit,
-    Lower,
-    Upper,
-    // Ctrl, < 0x20 || == DEL
-    // Print, = Graph || == ' '. NOT '\t' et cetera
-    Punct,
+const tIndex = enum(u4) {
+    Alpha, // Lower or Upper
+    Hex, // Digit or 'a'...'f' or 'A'...'F'
+    Space, // ' ', Form-feed, '\n', '\r', '\t', '\v' Vertical Tab
+    Digit, // '0'...'9'
+    Lower, // 'a'...'z'
+    Upper, // 'A'...'Z'
+    Punct, // ASCII and !DEL and !AlNum
```

Comment:
Nitpick: I recommend not making abbreviations like `Punctuation/Control -> Punct/Cntrl` — it may make it unnecessarily more difficult for non-native english speakers to understand the source.

Reply:
These abbreviations come from the C99 standard.

Reply:
Ah, my mistake 🎩

Reply:
On second thought, I still stand by my original suggestion because these are not constants which map to any C code values (from what I can tell). I feel like abbreviations like that are usually only good when you are exposing/wrapping around an already defined C value like `EPERM`, whereas a newly defined value should probably be `ErrorNotPermitted`. Anyway, not to get too sucked into a style discussion, I'm done :)